### PR TITLE
add 'polish' PR template

### PR DESCRIPTION
**Pull request by MichaelClifford**
## Related Issues and Dependencies

#31 

## This introduces a breaking change

- [x] Yes
- [ ] No

## This Pull Request implements

This PR, I moved the default PR template into a new directory `.git/PULL_REQUEST_TEMPLATE` and renamed it `SIMPLE_TEMPLATE`. I also created a new template for submitting finalized notebooks `POLISH_TEMPLATE`.  The point of these changes is to allow for multiple PR templates.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/POLISH_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+- [ ] There is a top markdown cell describing what the notebook does
+- [ ] There is a concluding markdown cell that summarizes the findings and outlines any next steps
+- [ ] All plots are the same size
+- [ ] All plots use seaborn (unless a specific functionality requires a different package)
+- [ ] All plots have a title, labeled axis, and some written description of the insights uncovered in the graph
+- [ ] Assuming correct environment, clicking "run all" works without error
+- [ ] All cells have been run and their output is displayed
+- [ ] Markdown headers are used to break up different sections
+- [ ] No error or warning messages are present in the outputs
+- [ ] All outputs are truncated to the minium size required for explanation. No more than 20 rows.
```
Comment (oindrillac) [inline .github/PULL_REQUEST_TEMPLATE/POLISH_TEMPLATE.md:24]: minium -> minimum

File: .github/PULL_REQUEST_TEMPLATE/POLISH_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+about: Finalize notebooks prior to publishing or promoting
+---
+
+## Related Issues and Dependencies
+
+…
+
+## The following checklist items have been completed for each notebook in this PR:
+
+- [ ] Proofread by other team member for grammar and spelling
```
Comment (oindrillac) [inline .github/PULL_REQUEST_TEMPLATE/POLISH_TEMPLATE.md:12]: This is cool. Can this step be made easier by using any tool like https://github.com/jupyterlab-contrib/spellchecker or https://jupyter-contrib-nbextensions.readthedocs.io/en/latest/nbextensions/spellchecker/README.html?

**Review by oindrillac — COMMENTED** [review]
Like this :+1:

**Review by tumido — COMMENTED** [review]
/lgtm

**Comment by isabelizimm** [comment]
I know this template is for notebooks in particular, but would it be useful to incorporate some sort of acceptance criteria on existence/grammar/spelling/standards for a README as well here?

**Review by aakankshaduggal — APPROVED** [review]
Looks good :rocket:

File: .github/PULL_REQUEST_TEMPLATE/POLISH_TEMPLATE.md (hunk 2 above)
Comment (MichaelClifford) [reply .github/PULL_REQUEST_TEMPLATE/POLISH_TEMPLATE.md:12]: @oindrillac yes, everyone should use a spellcheck extension for jupyter

File: .github/PULL_REQUEST_TEMPLATE/POLISH_TEMPLATE.md (hunk 1 above)
Comment (MichaelClifford) [reply .github/PULL_REQUEST_TEMPLATE/POLISH_TEMPLATE.md]: @oindrillac :+1:

**Comment by MichaelClifford** [comment]
> I know this template is for notebooks in particular, but would it be useful to incorporate some sort of acceptance criteria on existence/grammar/spelling/standards for a README as well here?

@isabelizimm good point, we could probably put some standards in place for README's but lets have that as a separate issue for the moment, and keep this one on notebooks.  : )

File: .github/PULL_REQUEST_TEMPLATE/POLISH_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+- [ ] Proofread by other team member for coherence and reproducibility (does the analysis make sense as you read through the code and markdown from top to bottom?)
```
Comment (chauhankaranraj) [inline .github/PULL_REQUEST_TEMPLATE/POLISH_TEMPLATE.md:13]: Just wanted to clarify, does "proofread for reproducibility" basically mean checklist item 9? Or is it something different?

**Review by Shreyanand — APPROVED** [review]
LGTM :+1:

File: .github/PULL_REQUEST_TEMPLATE/POLISH_TEMPLATE.md (hunk 3 above)
Comment (MichaelClifford) [reply .github/PULL_REQUEST_TEMPLATE/POLISH_TEMPLATE.md:13]: "proofread for reproduciblity" is more about, if you rerun a cell does the text around it still make sense or have the results changed in a way that causes the explanation and the results to be incongruous.

Item 9  "Assuming correct environment, clicking "run all" works without error" is just to check all the cells work and work in order.

Comment (chauhankaranraj) [reply .github/PULL_REQUEST_TEMPLATE/POLISH_TEMPLATE.md:13]: Gotcha, makes sense :+1:

**Review by chauhankaranraj — APPROVED** [review]
LGTM :rocket:

**Review by hemajv — APPROVED** [review]
/lgtm :ship:

**Review by isabelizimm — APPROVED** [review]

**Review by SankBad — APPROVED** [review]
LGTM :rocket:

**Review by tumido — APPROVED** [review]

**Pull request merged**